src/main/kotlin/org/rust/ide/core/overrideImplement/generateUtil.kt

```diff
@@ -87,13 +81,11 @@ fun insertNewTraitMembers(selected: Collection<RsTraitMemberChooserMember>, impl
             selected.mapNotNull { it.member as? RsConstant }
     )
     val lastMethodOrBrace = impl.functionList.lastOrNull() ?: impl.lbrace ?: return
-    runWriteAction {
-        impl.addRangeAfter(
-                templateImpl.lbrace?.nextSibling,
-                templateImpl.rbrace?.prevSibling,
-                lastMethodOrBrace
-        )
-    }
+    impl.addRangeAfter(
```

Comment:
I think I removed it because tests are running in write action (I set them up this way), so I forgot that in real work it can crash.